code/modules/organs/subtypes/machine.dm

```diff
@@ -75,13 +75,15 @@
 	..()
 	if(emp_counter)
 		emp_counter--
+		if(!emp_counter)
+			owner.remove_client_color(/datum/client_color/monochrome/damage)
 
 /obj/item/organ/internal/cell/emp_act(severity)
-	emp_counter += 30/severity
-	if(emp_counter >= 30)
-		owner.Paralyse(emp_counter/6)
-		to_chat(owner, "<span class='danger'>%#/ERR: Power leak detected!$%^/</span>")
-
+	emp_counter += 10 / severity
+	owner.add_client_color(/datum/client_color/monochrome/damage)
+	if(emp_counter >= 25)
+		owner.Paralyse(5)
```

Comment:
I figure a stronger ERP should floor you for more anyway. Play around with the values you get by dividing emp_counter.

Reply:
I find that taking damage across ALL your components is punishing enough, all the paralyse does is prevent IPCs from just yakety saxing away. It also forces them to drop their items. The damage to their limbs cause collapses and items to be dropped, so that justifies having a low paralyse count in the EMP act itself

Reply:
This literally only happens with severity 3 EMPs though